plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/dialects/SQLDialectDescriptor.java

```diff
@@ -0,0 +1,465 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.model.impl.dialects;
+
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.model.DBPImage;
+import org.jkiss.dbeaver.model.data.DBDInsertReplaceMethod;
+import org.jkiss.dbeaver.model.impl.AbstractContextDescriptor;
+import org.jkiss.dbeaver.model.sql.SQLDialect;
+import org.jkiss.dbeaver.model.sql.SQLDialectMetadata;
+import org.jkiss.utils.ArrayUtils;
+import org.jkiss.utils.BeanUtils;
+import org.jkiss.utils.CommonUtils;
+
+import javax.management.ReflectionException;
+import java.lang.reflect.Field;
+import java.util.*;
+
+/**
+ * SQLDialectDescriptor
+ */
+public class SQLDialectDescriptor extends AbstractContextDescriptor implements SQLDialectMetadata {
```

Comment:
Now we have two classes with the same name and similar methods. What is the purpose of having both of them?

Reply:
Mistake after refactoring, old one now removed